src/styles/card-block.scss

```diff
@@ -0,0 +1,97 @@
+$width: 200px;
+$main-text-color: #060b35;
+$link-buy-color: #00acdc;
+
+.card {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 12px;
+  background-color: #fff;
+
+  box-sizing: border-box;
+  width: $width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  padding: 33px 16px 17px;
```

Comment:
This should fix the bug in the tests
```suggestion
  padding: 32px 16px 16px;
```
<img width="311" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/8f18d987-92f9-4119-a175-6877814b7047">